docs/cloud/satellites/gha-runners.md

```diff
@@ -1,142 +1,103 @@
-- `admin:org_hook`
-- `admin:org`
-
-Fine grained token permissions: 
-- `organization_hooks:write`
-- `organization_self_hosted_runners:write`
-
-##### Repository integration
-For repository integrations, the user must be an admin of the repository and the token must have the following scopes/permissions:
-
-Personal access tokens (classic) scopes:
-- `admin:repo_hook`
-- `repo`
-
-Fine grained token permissions:
-- `repository_hooks:write`
-- `administration:write`
+{% hint style='info' %}
+Follow the [official docs](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/managing-your-personal-access-tokens) for detailed information on how to create a GitHub token, and make sure to set an expiration long enough, since the integration won't work after the token expires. 
+{% endhint %}
 
-### CLI
+### 2. Register the integration via CLI
+Create the integration using the `earthly github add` CLI command, passing the token created in the previous step. 
 
-`$ earthly github add --help`
+#### Organization integration
+``` 
+earthly github add \
+  --org <earthly_organization> \
+  --gh-org <github_organization> \
+  --gh-token <github_token>
+``` 
 
+#### Single repository integration
 ``` 
-NAME:
-earthly github add - Add GHA integration
-
-USAGE:
-earthly github add --org <org> --gh-org <github_organization> [--gh-repo <github_-repo>] --gh-token <github_token>
-
-DESCRIPTION:
-This command sets the configuration to create a new GitHub-Earthly integration, to trigger satellite builds from GHA (GitHub Actions).
-From the GitHub side, integration can be done at two levels: organization-wide and per repository.
-The provided token must have enough permissions to register webhooks and to create GitHub self hosted runners in those two scenarios.
-
-OPTIONS:
---org value       The name of the Earthly organization to set an integration with. Defaults to selected organization
---gh-org value    The name of the GitHub organization to set an integration with
---gh-repo value   The name of the GitHub repository to set an integration with
---gh-token value  The GitHub token used for the integration. Personal Access Token (classic) or fine grained tokens supported.
---help, -h        show help
-```
+earthly github add \
+  --org <earthly_organization> \
+  --gh-org <github_organization> \
+  --gh-repo <github_-repo> \
+  --gh-token <github_token>
+``` 
+
+### 3. Configure your satellites 
 
-## Satellite configuration
-This feature is currently disabled by default for satellites. It must be enabled in a per-satellite basis as follows:
+This feature needs to be enabled during satellite creation to be able to use it.
 
-### Managed (Earthly Cloud) Satellites
+#### Earthly-Cloud satellites
 Launch the satellite with the `enable-gha-runner` [feature-flag](https://docs.earthly.dev/earthly-cloud/satellites/managing#changing-feature-flags) enabled.
 ```
 earthly satellite --feature-flag enable-gha-runner <satellite-name>
```

Comment:
Are you missing the word `launch` in this command?

Reply:
good catch!